src/main.rs

```diff
@@ -100,7 +100,7 @@ async fn main() -> Result<(), LemmyError> {
       .configure(|cfg| images::config(cfg, &rate_limiter))
       .configure(nodeinfo::config)
       .configure(webfinger::config)
-      .service(actix_files::Files::new("/docs", "/app/documentation"))
+      .service(actix_files::Files::new("/docs", "/snap/lemmy/current/ui/dist/documentation"))
```

Comment:
Should this be added as a configuration option to the `Settings` struct?

Reply:
Most likely, yes.